src/mongo/db/auth/auth_index_d.cpp

```diff
@@ -100,7 +100,7 @@ Status verifySystemIndexes(OperationContext* txn) {
     std::vector<IndexDescriptor*> indexes;
     indexCatalog->findIndexesByKeyPattern(txn, v1SystemUsersKeyPattern, false, &indexes);
 
-    if (indexCatalog && !indexes.empty()) {
+    if (!indexes.empty()) {
```

Comment:
I presume you are removing this because if indexCatalog was null, line 99 above would be undefined behavior, and therefore we don't need the check?